pkg/operator/apiserver/controllerset/apiservercontrollerset.go

```diff
@@ -137,9 +141,39 @@ func (cs *APIServerControllerSet) WithClusterOperatorStatusController(
 		versionRecorder,
 		cs.eventRecorder,
 	)
+	cs.setCustomInertia()
 	return cs
 }
 
+// setCustomInertia uses controller names and status to set inertia appropriately.  It is called many times in order to
+// produce the same output regardless of the call order of adding control loops
+func (cs *APIServerControllerSet) setCustomInertia() {
+	if cs.clusterOperatorStatusController.controller == nil {
+		return
+	}
+
+	if len(cs.workloadConditionsPrefix) != 0 {
+		cs.clusterOperatorStatusController.controller = cs.clusterOperatorStatusController.controller.(*status.StatusSyncer).
+			WithDegradedInertia(status.MustNewInertia(
+				2*time.Minute,
+				// the workload deployment degraded condition happens when the number of available replicas does not
+				// match the desired state.  Assuming a PDB is present to ensure that we do not optionally dip below
+				// HA (not enforced here, but probably should be in the staticresourcecontroller check), we can safely
+				// allow being down a single replica for a considerable period of time.  Doing it here allows the
+				// detailed operator state to be fully correct (degraded instantly), while allowing graceful resolution
+				// in the majority of cases.
+				// The primary disadvantage is that we will no longer see signal in these operators when a single node
```

Comment:
@deads2k according to @ravisantoshgudimetla we don't install PDB in SNO.

Reply:
Correction: We don't intend to. I believe the PRs for creating PDBs for non-static components by @ingvagabund  did not take into account the cluster topology which might be a problem for future